File: Drivers/FSCC Drivers/windows/lib/fscc/c++/examples/tutorial.cpp
#include <iostream> // cout, endl
#include <cstdlib> // EXIT_SUCCESS, EXIT_FAILURE
#include <fscc.hpp> // Fscc::Port

int main(void)
{
    Fscc::Port p(0);

    // Send "Hello world!" text
    p.Write("Hello world!");

    // Read the data back in (with our loopback connector)
    std::cout << p.Read() << std::endl;

    return EXIT_SUCCESS;
}

File: Drivers/FSCC Drivers/windows/lib/fscc/net/examples/tutorial.cs
using Fscc;
using System;

public class Tutorial
{
    public static int Main(string[] args)
    {
        Fscc.Port p = new Fscc.Port(0);

        // Send "Hello world!" text
        p.Write("Hello world!");

        // Read the data back in (with our loopback connector)
        Console.WriteLine(p.Read(100));

        return 0;
    }
}